support/dynamics/gp/how-to-remove-a-company-that-is-no-longer-being-used.md

```diff
@@ -15,48 +15,26 @@ _Original KB number:_ &nbsp; 849249
 To remove a company that is no longer being used in Microsoft Dynamics GP, follow these steps:
 
 1. Follow the appropriate step:
-   - If you're using Microsoft Dynamics GP 10.0 or later, select **Tools** on the **Microsoft Dynamics GP** menu, point to **Utilities**, point to **System**, and then select **Delete Company**.
-   - If you're using Microsoft Dynamics GP 9.0, point to **Utilities** on the **Tools** menu, point to **System**, and then select **Delete Company**.
-   - If you're using Microsoft Business Solutions - Great Plains 8.0, select **Setup** on the **Tools** menu, point to **System**, and then select **Company**.
-   - If you're using a version that is earlier than Microsoft Business Solutions - Great Plains 8.0, select **System** on the **Setup** menu, and then select **Company**.
+   Select **Tools** on the **Microsoft Dynamics GP** menu, point to **Utilities**, point to **System**, and then select **Delete Company**.
 
 2. Select the **Company Name** lookup.
+
 3. Select the company that you want to delete, select **Select**, and then select **Delete**.
+
 4. Verify that the company database was removed. To do this, follow the steps for the database that you're using with Microsoft Dynamics GP.
 
-    - Microsoft SQL Server 2005/Microsoft SQL Server 2008/Microsoft SQL Server 2008 R2/Microsoft SQL Server 2012
+    - Microsoft SQL Server
 
       1. Start SQL Server Management Studio.
       2. Expand the instance of SQL Server for Microsoft Dynamics GP, and then expand **Databases**.
       3. Verify that the company database was removed. If the company database is listed, right-click the company database, and then select **Delete**.
 
-    - Microsoft SQL Server 2000
-
-      1. Start Enterprise Manager.
-      2. Expand the instance of SQL Server for Microsoft Dynamics GP, and then expand **Databases**.
-      3. Verify that the company database was removed. If the company database is listed, right-click the company database, and then select **Delete**.
-
-    - Microsoft SQL Server Desktop Engine (MSDE)
-
-      1. Start Support Administrator Console.
-      2. Run the following command to verify that the company database still exists.
-
-         ```sql
-         SELECT name FROM MASTER..SYSDATABASES
-         ```
-
-      3. If the company database is listed, run the following command to delete the database.
-
-         ```sql
-         DROP DATABASE XXX
-         ```
-
-         > [!NOTE]
+       > [!NOTE]
          > Replace **XXX** with the database name of the company that you want to delete.
```

Comment:
Hi @theley502, can you confirm whether we need to keep the NOTE in lines 32 and 33?